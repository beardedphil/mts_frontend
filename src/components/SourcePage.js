import React from 'react';

const SourcePage = () => {
    return (
        <div className="source-page">
            <h1>SourcePage Component</h1>
        </div>
    )
};

export default SourcePage;
