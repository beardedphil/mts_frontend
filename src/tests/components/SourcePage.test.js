import React from 'react';
import { shallow } from 'enzyme';
import SourcePage from '../../components/SourcePage';

test('should render SourcePage correctly', () => {
    const wrapper = shallow(<SourcePage />);
    expect(wrapper).toMatchSnapshot();
});
