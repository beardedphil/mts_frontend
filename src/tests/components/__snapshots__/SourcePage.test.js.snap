// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SourcePage correctly 1`] = `
<div
  className="source-page"
>
  <h1>
    SourcePage Component
  </h1>
</div>
`;
